neurodamus/core/configuration.py

```diff
@@ -1005,6 +1007,32 @@ def _spikes_sort_order(config: _SimConfig, run_conf):
                                  "BBP supports 'none' and 'by_time'")
 
 
+@SimConfig.validator
+def _coreneuron_direct_mode(config: _SimConfig, run_conf):
+    user_config = config.cli_options
+    direct_mode = user_config.coreneuron_direct_mode
+    if direct_mode:
+        if config.use_neuron:
+            logging.warning("--coreneuron-direct-mode not valid for NEURON, will do nothing")
+            direct_mode = False
```

Comment:
IMO we can and probably want to error out so the user can fix his inout. Discarding this CLI option because the Simulator is Neuron sounds dangerous, maybe the user really wanted CoreNeuron

Reply:
Ok, I've changed to raise an exception in this case.